jekyll/userguide.md

```diff
@@ -67,7 +67,7 @@ This view shows all assets for the selected portfolio on a map. It also acts as
 #### Estimated Loss View
 
 <p align="center">
-<img height="300" src="assets/images/Estvalue.png">
+<img height="300" src="assets/images/Estimatedvalue.png">
```

Comment:
This should be `estvalue.png`. I suspect you have another file locally with a different name that is not tracked in git